.claude/skills/twelve-factor/agents/deployment-validator.sh

```
@@ -0,0 +1,175 @@
+        echo "parity_violation"
+    else
+        echo "ok"
+    fi
+}
+
+check_ci_cd() {
+    log "Checking CI/CD configuration..."
+
+    local ci_files=()
+
+    [ -f "$PROJECT_ROOT/.github/workflows"/*.yml ] && ci_files+=("github_actions")
```

Shellcheck (0.11.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)